system_reports/osx-xcode-10.0.x.log

```diff
@@ -20,7 +20,65 @@
 * Java: openjdk version "1.8.0_222"
 OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_222-b10)
 OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)
-* Flutter: Flutter 1.9.1+hotfix.2 • channel stable • https://github.com/flutter/flutter.git
+Unhandled exception:
```

Comment:
It looks like something went wrong with Flutter on this stack.

Reply:
Thanks @koral for spotting this, we will look into it, before rotating our the new stacks updates